### ISSUE TITLE
Error coming from amp-accordion

### ISSUE THREAD

**Issue by sparhami** [ampproject/amphtml#23097]
```
Error: An invalid or illegal string was specified.
    at observe (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/extensions/amp-accordion/0.1/amp-accordion.js:159:6)
    at forEach (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/extensions/amp-accordion/0.1/amp-accordion.js:87:4)
    at buildCallback (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/custom-element.js:513:18)
    at call (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/node_modules/promise-pjs/promise.js:502:6)
    at doResolve (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/node_modules/promise-pjs/promise.js:35:2)
    at https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/custom-element.js:510:38
    at build (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/service/resource.js:337:11)
    at build (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/service/resources-impl.js:658:20)
    at buildResourceUnsafe_ (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/service/resources-impl.js:598:8)
```

```
Error: SyntaxError: DOM Exception 12
    at section (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/extensions/amp-accordion/0.1/amp-accordion.js:159:29)
    at https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/extensions/amp-accordion/0.1/amp-accordion.js:87:27
    at https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/custom-element.js:513:18
    at https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/custom-element.js:510:50
    at https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/service/resource.js:337:11
    at https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/service/resources-impl.js:657:47
    at buildResourceUnsafe_ (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/service/resources-impl.js:598:8)
    at buildOrScheduleBuildForResource_ (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/service/resources-impl.js:724:4)
    at https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/custom-element.js:432:35
    at impl (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/custom-element.js:907:30)
    at https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/custom-element.js:401:8
    at toClass (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/service/custom-element-registry.js:73:41)
    at win (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/service/extensions-impl.js:341:29)
    at ampdoc (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/service/extensions-impl.js:323:40)
    at style (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/style-installer.js:68:9)
    at ampdoc (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/service/extensions-impl.js:320:8)
    at ampdoc (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/service/extensions-impl.js:305:27)
    at ampdoc (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/service/extensions-impl.js:390:16)
    at https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/service/extensions-impl.js:304:23
    at TAG (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/extensions/amp-accordion/0.1/amp-accordion.js:38:12)
    at https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/extensions/amp-accordion/0.1/amp-accordion.js:629:26
    at arg (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/service/extensions-impl.js:170:14)
    at fnOrStruct (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/runtime.js:187:39)
```

There are two different stack traces that I think these are the same issue, but I am not sure.

go/ampe/CP-WoIfG56zOPQ
go/ampe/CK_bk82bgs3HRg

**Comment by sparhami** [comment]
Also likely related:

```
Error: failed to build: amp-accordion#20 SyntaxError
at (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/log.js:565)
at (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/log.js:241)
at (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/log.js:256)
at isBlockedByConsent (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/service/resource.js:366)
at maybeReportErrorOnBuildFailure (https://raw.githubusercontent.com/ampproject/amphtml/1906242222460/src/service/resource.js:353)
```

go/ampe/CJi8jLXxhJ_v6gE

**Comment by devnullzero** [comment]
Probably related... :

```
Powered by AMP ⚡ HTML – Version 1907022322580 <snip> v0.js:572

    SyntaxError: DOM Exception 12   v0.js:142
        ag					       v0.js:142
        (anonymous function)	       v0.js:268
        (anonymous function)	       v0.js:43
        a					       v0.js:44

    SyntaxError: DOM Exception 12  v0.js:142

	<snip>

    failed to build: amp-accordion#19: SyntaxError: DOM Exception 12 	v0.js:142
        ag                                                            v0.js:142
        (anonymous function)                              v0.js:541
        (anonymous function)
        error                                                         v0.js:19
        maybeReportErrorOnBuildFailure              v0.js:243
        (anonymous function)                               v0.js:242
        (anonymous function)                               v0.js:43
        a                                                               v0.js:44

    failed to build: amp-accordion#20: SyntaxError: DOM Exception 12 	v0.js:142

	<snip>

9SyntaxError: DOM Exception 12: An invalid or illegal string was specified. 	amp-accordion-0.1.js:17

```
Using an old browser:

```
Mozilla/5.0 (BB10; Touch) AppleWebKit/537.35+ (KHTML, like Gecko) Version/10.3.3.3216 Mobile Safari/537.35+
```

**Comment by itsmsc** [comment]
Probably related... :
`Error: An invalid or illegal string was specified.
    at https://cdn.ampproject.org/v0/amp-accordion-0.1.js:17:342
    at Array.forEach (native)
    at S.push.f.S.buildCallback (https://cdn.ampproject.org/v0/amp-accordion-0.1.js:15:200)
    at https://cdn.ampproject.org/v0.js:267:336
    at Cc (https://cdn.ampproject.org/v0.js:45:283)
    at new zc (https://cdn.ampproject.org/v0.js:39:901)
    at c.b.build (https://cdn.ampproject.org/v0.js:267:91)
    at Ti.f.build (https://cdn.ampproject.org/v0.js:242:168)
    at Oj (https://cdn.ampproject.org/v0.js:325:326)
    at Nj (https://cdn.ampproject.org/v0.js:324:249)`

Also using a old browser:
`Mozilla/5.0 (Linux; Android 4.4.2; Android SDK built for x86 Build/KK) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/30.0.0.0 Mobile Safari/537.36`

**Comment by sparhami** [comment]
Closed via #23328
